review/developer/small-prs.md

```diff
@@ -1,162 +1,162 @@
     change.
 -   **Less blocking on reviews.** Sending self-contained portions of your
     overall change allows you to continue coding while you wait for your current
-    CL in review.
--   **Simpler to roll back.** A large CL will more likely touch files that get
-    updated between the initial CL submission and a rollback CL, complicating
-    the rollback (the intermediate CLs will probably need to be rolled back
+    PR in review.
+-   **Simpler to roll back.** A large PR will more likely touch files that get
+    updated between the initial PR submission and a rollback PR, complicating
+    the rollback (the intermediate PRs will probably need to be rolled back
     too).
 
 Note that **reviewers have discretion to reject your change outright for the
 sole reason of it being too large.** Usually they will thank you for your
 contribution but request that you somehow make it into a series of smaller
 changes. It can be a lot of work to split up a change after you've already
 written it, or require lots of time arguing about why the reviewer should accept
-your large change. It's easier to just write small CLs in the first place.
+your large change. It's easier to just write small PRs in the first place.
 
 ## What is Small? {#what_is_small}
 
-In general, the right size for a CL is **one self-contained change**. This means
+In general, the right size for a PR is **one self-contained change**. This means
 that:
 
--   The CL makes a minimal change that addresses **just one thing**. This is
+-   The PR makes a minimal change that addresses **just one thing**. This is
     usually just one part of a feature, rather than a whole feature at once. In
-    general it's better to err on the side of writing CLs that are too small vs.
-    CLs that are too large. Work with your reviewer to find out what an
+    general it's better to err on the side of writing PRs that are too small vs.
+    PRs that are too large. Work with your reviewer to find out what an
     acceptable size is.
--   The CL should [include related test code](#test_code).
--   Everything the reviewer needs to understand about the CL (except future
-    development) is in the CL, the CL's description, the existing codebase, or a
-    CL they've already reviewed.
+-   The PR should [include related test code](#test_code).
+-   Everything the reviewer needs to understand about the PR (except future
+    development) is in the PR, the PR's description, the existing codebase, or a
+    PR they've already reviewed.
 -   The system will continue to work well for its users and for the developers
-    after the CL is checked in.
--   The CL is not so small that its implications are difficult to understand. If
-    you add a new API, you should include a usage of the API in the same CL so
+    after the PR is merged.
+-   The PR is not so small that its implications are difficult to understand. If
+    you add a new API, you should include a usage of the API in the same PR so
```

Comment:
For me this is an important section and addresses a few things that came up in the discussion.
`1 PR != 1 feature`
So this makes the debate about squash or not squash less relevant, and a matter of personal taste (given you don't rewrite the history of a shared branch like master)